subiquity/server/controllers/filesystem.py

```diff
@@ -160,6 +160,10 @@ def __init__(self, app):
         self._role_to_device: Dict[str: _Device] = {}
         self._device_to_structure: Dict[_Device: snapdapi.OnVolume] = {}
         self.use_tpm: bool = False
+        self.locked_probe_data = False
```

Comment:
Since I can only review the Python and only a bit of the logic, a small thing: per the style here, shouldn't this be typed:
`self.locked_probe_data: bool = False` ?

Reply:
Good catch, thanks! I included this change in a follow-on PR: https://github.com/canonical/subiquity/pull/1666